routes/manual.tsx

```diff
@@ -244,55 +244,57 @@ function ToC({
           const active = path === `/${slug}`;
           return (
             <li key={slug}>
-              <input
-                type="checkbox"
-                id={slug}
-                class={tw`hidden checked:siblings:even:children:first-child:rotate-90 checked:siblings:last-child:block`}
-                checked={active || path.startsWith(`/${slug}/`)}
-                disabled={!entry.children}
-              />
-
-              <label
-                htmlFor={slug}
-                class={tw`flex items-center gap-2 px-2.5 py-2 rounded-md block ${
-                  active ? "link bg-ultralight" : "hover:text-gray-500"
-                } font-semibold` + (active ? " toc-active" : "")}
+              <details
+                open={active}
+                class="rotate-svg"
+                // @ts-ignore onToggle does support strings
+                onToggle={`event.currentTarget.firstChild.firstChild.setAttribute("aria-label", (event.currentTarget.open ? "close" : "open") + " section ${entry.name}")`}
               >
-                <Icons.TriangleRight
-                  class={entry.children ? "" : "invisible"}
-                />
-                <a href={`/manual${version ? `@${version}` : ""}/${slug}`}>
-                  {entry.name}
-                </a>
-              </label>
+                <summary
+                  class={tw`flex items-center gap-2 px-2.5 py-2 rounded-md ${
+                    active ? "link bg-ultralight" : "hover:text-gray-500"
+                  } font-semibold` + (active ? " toc-active" : "")}
+                >
+                  <Icons.TriangleRight
+                    aria-label={`open section ${entry.name}`}
+                    onKeyDown="if (event.code === 'Space' || event.code === 'Enter') { this.parentElement.click(); event.preventDefault(); }"
+                    tabindex={0}
+                    class={"cursor-pointer" +
+                      (entry.children ? "" : " invisible")}
+                  />
```

Comment:
This can go entirely if you make `<summary>` a `display: list-item` instead of a `display: flex`.

Reply:
the built-in triangle is worse, but ok...